README.md

```diff
@@ -1,4 +1,6 @@
 # Material Motion Android Runtime
+[![Build Status](https://circleci.com/gh/material-motion/material-motion-runtime-android.svg?style=shield)](https://circleci.com/gh/material-motion/material-motion-runtime-android)
```

Comment:
@pingpongboss - someone on the `material-motion` team would need to log into `circleci` and grant it access to build the repo. Once the PR is merged to develop, those badges should work ok. https://github.com/seanoshea/BluePlaquesLondon-Android/blob/develop/README.md is one example of how this works for other projects.